### PR TITLE
Remove administrative information redundant with draft-ietf-git-github-wg-configuration

### PR DESCRIPTION
**Pull request by chris-wood**
I was tempted to remove the entire section on administrative policies, though left it as is. @martinthomson, what do you think?

### CODE REVIEW

File: draft-ietf-git-using-github.md [hunk 1]
```diff
@@ -167,29 +166,8 @@ and any Working Group Secretary.  Administrator access is preferable, since this
...
-resources.  It is desirable for those to be accessible via the Working Group's
-datatracker page.  For example, this might be via URLs listed in the More Info
-section on the Working Group Charter page.
-
-The IETF MAY decide to backup information associated with a Working Group's
-organization periodically.  This decision can be made differently per Working
-Group in consultation with the responsible Area Director.
-
+Details about creating organizations adhering to these guidelines can be found
+in {{!I-D.ietf-git-github-wg-configuration}}.
```
Comment (martinthomson) [inline draft-ietf-git-using-github.md]:
```suggestion
in {{!GIT-CONFIG=I-D.ietf-git-github-wg-configuration}}.
```

**Review by martinthomson — APPROVED** [review]
I was thinking of being a little more aggressive about removing some of the text, with matching PRs for the other work, but this works.

**Comment by chris-wood** [comment]
> I was thinking of being a little more aggressive about removing some of the text

Yeah, I figured it best to start small. Let me know if you need anything else before merging!

**Comment by martinthomson** [comment]
@chris-wood, Interesting that the circle CI build ran on the merge, but not the PR.  I didn't see any build failures reported before it merged, but I just got the email reporting a missed reference.

Maybe there are some tweaks you need to apply over there as owner of the org.

**Comment by chris-wood** [comment]
> Maybe there are some tweaks you need to apply over there as owner of the org.

Yeah, I'll fix it. Thanks for letting me know!

**Pull request merged**